python/teca_py_alg.i

```diff
@@ -307,6 +325,13 @@
 from teca_python_algorithm import *
 %}
 
+/***************************************************************************
+ model_segmentation
+ ***************************************************************************/
+%pythoncode %{
```

Comment:
http://cdash.hpcvis.com/testDetails.php?test=150740&build=6394
deal with this error here, you can wrap the import in a try, except pass